client/rest-high-level/src/main/java/org/opensearch/client/indices/CloseIndexResponse.java

```diff
@@ -107,8 +107,12 @@ public static class IndexResult {
         static {
             PARSER.declareBoolean(optionalConstructorArg(), new ParseField("closed"));
             PARSER.declareObject(optionalConstructorArg(), (p, c) -> {
-                XContentParserUtils.ensureExpectedToken(XContentParser.Token.START_OBJECT, p.currentToken(), p);
-                XContentParserUtils.ensureExpectedToken(XContentParser.Token.FIELD_NAME, p.nextToken(), p);
+                org.opensearch.core.xcontent.XContentParserUtils.ensureExpectedToken(
```

Comment:
Seems like the full import is there already:
```suggestion
                XContentParserUtils.ensureExpectedToken(
```